<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DenRoze3</class>
 <widget class="QMainWindow" name="DenRoze3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DenRoze3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QWidget" name="stock_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="login_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QWidget" name="bills_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="menu_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QLabel" name="main_menu_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>153</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Main menu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stock_button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>153</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Sklad</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bills_button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>153</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Prodej</string>
       </property>
      </widget>
      <widget class="QPushButton" name="orders_button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>153</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Objednavky</string>
       </property>
      </widget>
      <widget class="QPushButton" name="write_button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>153</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Zapsat zmeny</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QWidget" name="orders_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
